examples/js/loaders/ColladaLoader.js

```diff
@@ -1328,6 +1332,54 @@ THREE.ColladaLoader.prototype = {
 			}
 
 		}
+		
+		function parseEffectExtra( xml ) {
+ 
+            		var data = {};
+ 
+            		for ( var i = 0, l = xml.childNodes.length; i < l; i ++ ) {
+ 
+                		var child = xml.childNodes[ i ];
+ 
+                		if ( child.nodeType !== 1 ) continue;
+ 
+                		switch ( child.nodeName ) {
+ 
+                    		case 'technique':
+                        		data.technique = parseEffectExtraTechnique( child );
+                        		break;
+ 
+                		}
+ 
+            		}
+ 
+            		return data;
+ 
+        	}
+        
+        	function parseEffectExtraTechnique( xml ) {
+ 
+            		var data = {};
+ 
+            		for ( var i = 0, l = xml.childNodes.length; i < l; i ++ ) {
+ 
+                		var child = xml.childNodes[ i ];
+ 
+                		if ( child.nodeType !== 1 ) continue;
+ 
+                		switch ( child.nodeName ) {
+ 
+                    		case 'double_sided':
+                    			data[ child.nodeName ] = parseInt( child.textContent );
```

Comment:
What are the possible values of `double_sided`?

Reply:
It's an integer that can be 1 or 0
though the element will probably won't ever be added if it's 0

I quickly exported a collada file with blender. Two cubes where one cube is double sided.

[cube_ds.dae.zip](https://github.com/mrdoob/three.js/files/1696017/cube_ds.dae.zip)

Reply:
Because of the unpredictable nature of exporters i would explicitly test for a `1` value. Otherwise we might set `DoubleSide` when it's actually not requested...

Reply:
Yep very reasonable, so... I should add this, or do you add this? ... I'm new to pull requests sorry

Reply:
Give it a try 😉

Reply:
Voila? hehe